src/components/ProjectForm/useFetch.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+function useFetch(url) {
+  const [data, setData] = useState();
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const { data } = await axios.get(url);
+        setData(data);
+      } catch (err) {
+        setError(err);
+      }
+    }
+    fetchData();
+  }, [url]);
+
+  return {
+    data,
+    error,
+  };
+}
+
+export default useFetch;
```

Comment:
There's an existing file for this. ProjectsAPI.js .  Create your function inside that one
![image](https://user-images.githubusercontent.com/85913826/126727479-e4864aff-6b1d-4158-a5f0-936616fe1586.png)

Reply:
That ins´t in the file, 
![image](https://user-images.githubusercontent.com/85940891/126727889-bb5af373-cf24-442d-8215-ccf1f2bdca7f.png)